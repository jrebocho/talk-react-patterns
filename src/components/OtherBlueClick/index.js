import React from 'react'
import Stateless from '../Stateless'
import BlueClickRenderProp from '../BlueClickRenderProp'

const OtherBlueClick = () => (
  <BlueClickRenderProp render={on => <Stateless on={on} />} />
)

export default OtherBlueClick
