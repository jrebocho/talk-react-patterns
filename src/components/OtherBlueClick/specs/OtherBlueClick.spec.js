import React from 'react'
import { shallow } from 'enzyme'
import OtherBlueClick from '../../OtherBlueClick'

describe('<OtherBlueClick />', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<OtherBlueClick />)
    expect(wrapper).toMatchSnapshot()
  })
})
