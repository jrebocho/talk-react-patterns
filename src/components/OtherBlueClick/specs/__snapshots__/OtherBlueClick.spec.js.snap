// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<OtherBlueClick /> renders correctly 1`] = `
<BlueClickRenderProp
  render={[Function]}
/>
`;
